app.py

```diff
@@ -137,7 +137,6 @@ def main():
                 user = datastore.Entity(key=key)
                 user.update(
                     {
-                        "joined": datetime.datetime.utcnow(),
```

Comment:
Did you mean to remove this?

Reply:
Yes. It made the json.dumps() not work, and was a useless field.

Reply:
I think it should be OK now that you are building your own dictionary at line 396. That said, if you still want to remove it, that seems fine.

Reply:
It does work when you leave joined in.